packages/react/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "@apoyo/react",
+  "version": "0.0.0",
+  "description": "React utility library",
+  "main": "src/index.js",
+  "module": "es6/index.js",
+  "typings": "lib/index.d.ts",
+  "author": "Julien Capellari <julien.capellari@gmail.com>",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/neoxia/apoyo"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "homepage": "https://github.com/neoxia/apoyo/tree/master/packages/react",
+  "scripts": {
+    "test": "npx jest --clearCache && jest --verbose",
+    "clean": "npx rimraf ./dist",
+    "prebuild": "npm run clean",
+    "build": "npx tsc -p ./tsconfig.common.json && npx tsc -p ./tsconfig.es.json && npx ts-node ../../scripts/build"
+  },
+  "dependencies": {
```

Comment:
Maybe make react / react-dom peerDependencies?
This way, the user has more control on which react version he wishes to install.